resources/models.py

```diff
@@ -252,6 +263,49 @@ def __str__(self):
         return self.safe_translation_getter("name", super().__str__())
 
 
+class AbstractAreaMap(UUIDModel):
+    map_file = models.FileField(
+        upload_to=get_map_resource_media_folder,
+        storage=OverwriteStorage(),
+        verbose_name=_("map file"),
+        null=True,
```

Comment:
Should not be nullable or blankable, since the entire `HarborMap` or  `WinterStorageAreaMap ` object is about this field.